python/requirements_test.txt

```diff
@@ -68,6 +68,7 @@ testfixtures==7.0.0
 werkzeug==2.1.2
 xlrd==2.0.1
 memray; platform_system != "Windows" and sys_platform != "darwin"
+jupyter
```

Comment:
should we be pinning version here, cc @krfricke

Reply:
actually we don't need this anymore. I will remove it..